### DISCUSSION TITLE
Only enable markdown styling for lsp and cmp

### DISCUSSION THREAD

**Discussion by seblyng** [folke/noice.nvim#323] [Q&A]
Hi! I have been trying this plugin out and it's really impressing what you manage to do with the UI! However, I am currently only interested in styling the markdown in cmp and lsp by overriding like:

```lua
lsp = {
    override = {
        ["vim.lsp.util.convert_input_to_markdown_lines"] = true,
        ["vim.lsp.util.stylize_markdown"] = true,
        ["cmp.entry.get_documentation"] = true,
    },
},
```

Are there any chance that this may get it's own plugin? Or an easy way to turn everything off except this? I tried to extract this out myself, but it looks tightly coupled with a lot of things going on, and I couldn't find an easy way to only achieve this.

Thanks!

**Comment by folke** [comment]
not possible. You can easily disable other components though
